cmd/libs/go2idl/client-gen/generators/generator-for-type.go

```diff
@@ -0,0 +1,205 @@
+/*
```

Comment:
I make this a separate file because we will have other generator, like the one for the group client in #17699